lib/lifecycle.js

```diff
@@ -175,6 +177,12 @@ async function runBenchmark(
 			},
 			benchmarkMode,
 		);
+
+		if (opsSecPerRun) {
```

Comment:
This should be placed after line 191

```diff 
const opsSec = totalIterations / (totalTimeSpent / timer.scale);
+ opsSecPerRun.push(opsSec)
```

Reply:
opsSec is already an average. We should store the opsSec of each benchmark run. Otherwise, this array will be bloated by numbers.

Reply:
> We should store the opsSec of each benchmark run

well that is whats happening with the current change, here are the results with my proposal and what you are recommending:

both benchmarks have this config: `{ repeatSuite: 5, minSamples: 10, minTime: 0.1, maxTime: 1 }`

this is the result with my proposal:
```
    opsSec: 69265522.16431555,
    opsSecPerRun: [
      69906546.27213925,
      69322349.58518846,
      67852871.05875228,
      69748757.75946592,
      69496837.202617
    ]
```
:point_up: returns each run, 5 in total.

and this is  with your recommendation:
```
    opsSec: 67969820.81042846,
    opsSecPerRun: [ 67969820.81042846 ]
```
:point_up: it only stores one opsSec, the averaged total